example/ios/WindowViewExample/AppDelegate.m

```diff
@@ -37,6 +37,15 @@ @interface RNWindow : UIWindow
 
 @implementation RNWindow
 
+- (UIView *) hitTest:(CGPoint)point withEvent:(UIEvent *)event
```

Comment:
Why is it here? `RNWindow` is the window containing the application so won't it result in destroying the behavior of the components?

Reply:
I just tested without adding this `hitTest` here and things are working well - `RNWindowView` passes touches though even when it's shown.